data/sql/updates/pending_db_world/rev_1680463160388009300.sql

```diff
@@ -0,0 +1,11 @@
+-- Remove old/deprecated bytes2 flags.
+UPDATE `creature_addon` SET `bytes2` = 1 WHERE `bytes2` IN (4092,4096,4097,256,257,301993985,318771201,285216769);
+UPDATE `creature_addon` SET `bytes2` = 0 WHERE `bytes2` IN (733,141313,234885121,251662337);
+UPDATE `creature_addon` SET `bytes2` = 2 WHERE `bytes2` IN (258,4098);
```

Comment:
4096 -> 0

Reply:
256 -> 0 going from bitmask

Reply:
I think 733 -> 1
(1, 4, 8, 16, 64, 128, 512)

Reply:
141313 is on 4 mobs (1 of which is wrong and gonna get deleted) beasts